inductiva/fluids/post_processing/openfoam.py

```diff
@@ -46,28 +46,44 @@ def __init__(self, sim_output_path: types.Path):
 
         self.sim_output_path = sim_output_path
         self.full_output = self.inspect_output()
+        print(self.full_output)
 
     def inspect_output(self):
-        """Inspect the output of the simulation."""
+        """Inspect the output of the simulation.
+        
+        Return True if the simulation output contains more files than
+        the default ones. This is used to determine if the post-processing
+        was computed on the backend or not."""
```

Comment:
If the function returns this, then it should be named `is_full()` or something similar. `inspect_output` suggests that you want to print the output file names, plot them quickly or something of the sort.